tools/piegen/piegen.js

```diff
@@ -23,21 +23,75 @@ function getRandomNumber(min, max) {
   return Math.max(min, Math.round((Math.random() * max)));
 }
 
-const MIN_WEIGHT = 300; // In grams.
-const MAX_WEIGHT = 1000; // In grams.
-const MAX_PRICE = 800;
+const ADJACTIVES = [
+  'mouth-watering',
+  'delicious',
+  'tasty',
+  'crafty',
+  'well-done'
+];
+
+const NOUNS = [
+  'combination',
+  'pie',
+  'composition'
+];
+
+const VERBS = [
+  'handcrafted from',
+  'filled with',
+  'served with'
+]
+
+function generateDescription(pie) {
+  const adjIndex = getRandomNumber(0, ADJACTIVES.length - 1);
+  const adj1 = ADJACTIVES[adjIndex];
+  const adj2 = ADJACTIVES[(adjIndex+1)%ADJACTIVES.length];
+  const adj3 = ADJACTIVES[(adjIndex+2)%ADJACTIVES.length];
+  const noun = NOUNS[getRandomNumber(0, NOUNS.length - 1)];
+  const verb = VERBS[getRandomNumber(0, VERBS.length - 1)];
+  const filling = pie.filling;
+  const dough = pie.dough;
+  return `A ${adj1} and ${adj2} ${noun}, ${verb} ${filling} and wrapped in a ${adj3} ${dough}`;
```

Comment:
I love how all your adjectives start with a consonant, so you don't have to worry about 'a' vs 'an' ;)

Reply:
Yeah, you caught me;)